second_response_to_reviewers.tex

```diff
@@ -0,0 +1,366 @@
+more evolutionary robust version of Robert Axelrod's original tournament.
+
+\section{Comments from third reviewer}
+
+\begin{verbatim}
+    I saw no significant contribution in the field of evolutionary computation
+    itself. The whole paper would have been a great *experimental section* after
+    some new idea or scheme.
+\end{verbatim}
+
+We have justified the contribution in many ways but agree that this paper is a
+strong numerical experimental paper (as evidenced by the title) following the
+novel idea of creating tournaments with Moran processes as ways to score pairs of
+strategies.
+
+\begin{verbatim}
+Indeed, results appears sensible, when not obvious. Collective strategies
+perform well, the usual suspects are hard to invade. However, generally
+speaking, the main problem is that with IPD is extremely hard to infer
+meaningful data from the mere statistical results, as it would be hard to draw
+conclusions about the "best opening" looking at students playing chess. Indeed,
+such an analysis makes perfect sense as soon as one restricts the domain to a
+specific grandmaster, or a specific top-level tournament.
+\end{verbatim}
+
+A lot of the original and contemporary research in this area uses the IPD to
+show why/how
+cooperative behaviour may evolve. This has often been through large numerical
+analyses such as these. However, none of them (as discussed in the literature
+review) consider Moran processes on this scale. Furthermore, one of our main
+conclusions/findings is that handshake mechanisms can also appear through
+evolutionary processes and in particular that these can explain the emergence of
+cooperation.
+
+\begin{verbatim}
+    The *Moran Processes* should be briefly introduced, let alone mentioned in
+    the abstract before explaining how their *fixation probabilities* have been
+    obtained.
+\end{verbatim}
+
+The Moran process is briefly introduced in section A. It is also described in
+Fig 1, Algorithm 2 and also mathematically throughout equations (1) - (7).
+Furthermore numerous background references are given.
+
+\begin{verbatim}
+The concept of *fixation probabilities* should be properly introduced, or, more
+simply, authors can talk about *the probability with which a given strategy
+takes over a population* without mentioning the specific term.
+\end{verbatim}
+
+We have added a description of this to the main text.
+
+\begin{verbatim}
+As a good journal paper could be read several years after being written, authors
+should avoid relative expressions like "recent" or recently", and use absolute
+timings, such as "in 2012" or "in the beginning of 2010s".
+\end{verbatim}
+
+We have modified this terminology.
+
+\begin{verbatim}
+    Population sizes range from 2 to 14. While 2 is an obvious choice, being the
+    smallest possible *population*, the use of 14 is puzzling. Twice the days in
+    a week?
+\end{verbatim}
+
+There needs to be an upper bound, 14 was chosen as the highest number reasonable
```

Comment:
Additionally, the top strategies stabilize quickly and indeed the most interesting values of N are the smaller ones (and reference the figure).